src/hook_generated.h

```diff
@@ -0,0 +1,15 @@
+#ifndef HOOK_GENERATED_H
+#define HOOK_GENERATED_H
+#include <stdio.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+void hook_generated(FILE* hook_log);
```

Comment:
Do you think we should put this directly into hook.h? Probably cleaner

Reply:
Agree 👍

Reply:
I have pushed a new patch with this change